E275: false positive with f-strings in Python 3.12

```
[adrian@eluvian:/tmp]> ./testenv-312/bin/flake8 --version
6.1.0 (mccabe: 0.7.0, pycodestyle: 2.11.0, pyflakes: 3.1.0) CPython 3.12.0 on Linux

[adrian@eluvian:/tmp]> ./testenv-311/bin/flake8 --version
6.1.0 (mccabe: 0.7.0, pycodestyle: 2.11.0, pyflakes: 3.1.0) CPython 3.11.6 on Linux

[adrian@eluvian:/tmp]> cat util.py
def bug(wtf):
    return f'in{wtf}'

[adrian@eluvian:/tmp]> ./testenv-311/bin/flake8 util.py
[adrian@eluvian:/tmp]> ./testenv-312/bin/flake8 util.py
util.py:2:16: E275 missing whitespace after keyword
```

For some reason it seems to expect a space before the `{...}` inside the f-string. of course this makes no sense, since that would change the resulting string.

--- a/pycodestyle.py
+++ b/pycodestyle.py
@@ -490,6 +490,7 @@ def missing_whitespace_after_keyword(logical_line, tokens):
         # appear e.g. as "if x is None:", and async/await, which were
         # valid identifier names in old Python versions.
         if (tok0.end == tok1.start and
+                tok0.type == tokenize.NAME and
                 keyword.iskeyword(tok0.string) and
                 tok0.string not in SINGLETONS and
                 not (tok0.string == 'except' and tok1.string == '*') and

--- a/testing/data/python36.py
+++ b/testing/data/python36.py
@@ -1,2 +1,3 @@
 #: Okay
 f'{hello}:{world}'
+f'in{x}'

